### PR TITLE
MA-3786: Updated runtime node version to 8.10 and ProjectImage to 8.11.0

### PR DESCRIPTION
**Pull request by bhargavapratik**
MA-3786: Updated runtime node version to 8.10 and ProjectImage to 8.11.0.

### CODE REVIEW

File: template.yaml [hunk 1]
```diff
@@ -80,7 +80,7 @@ Resources:
     Properties:
       FunctionName: SungardAS-AWSConfig
       Handler: index.handler
-      Runtime: nodejs4.3
+      Runtime: nodejs8.10
```
Comment (SonalOjha) [inline template.yaml:83]: Make this a parameter to the template

**Review by SonalOjha — CHANGES_REQUESTED** [review]

File: template.yaml (hunk 1 above)
Comment (bhargavapratik) [reply template.yaml:83]: Runtime cannot be a Parameter. still if we need to try this , then we can do while we move code to BitBucket.

Comment (RajkumarShivage) [reply template.yaml:83]: We will take care of this when we move code to BitBucket.

**Review by RajkumarShivage — APPROVED** [review]
LGTM

File: template.yaml (hunk 1 above)
Comment (bhargavapratik) [reply template.yaml:83]: MA-4154 Jira for taking Runtime as Parameter.

**Review by SonalOjha — APPROVED** [review]
With changes getting addresses in MA-4154 approving the PR.

**Pull request merged**